integration/base/fixable/a/a/a.ts

```diff
@@ -0,0 +1,6 @@
+import path from 'path';
```

Comment:
For those without Lingersvision:

```suggestion
// Imports out-of-order
import path from 'path';
```